src/wizard/javascript/replay-onboarding/astro/2.configure.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Astro
+doc_link: https://docs.sentry.io/platforms/javascript/guides/astro/session-replay/
+support_level: production
+type: language
+---
+
+#### Configure
+
+There are several privacy and sampling options available, all of which can be set using the `integrations` constructor. Learn more about configuring Session Replay by reading the [configuration docs](https://docs.sentry.io/platforms/javascript/session-replay/). Add the following to your SDK config:
+
+```javascript
+import * as Sentry from "@sentry/astro";
+import { Replay } from "@sentry/replay";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+
+  // This sets the sample rate at 10%. You may want this to be 100% while
+  // in development, then sample at a lower rate in production.
+  replaysSessionSampleRate: 0.1,
+  // If the entire session is not sampled, use the below sample rate to sample
+  // sessions when an error occurs.
+  replaysOnErrorSampleRate: 1.0,
+
+  integrations: [new Replay()],
```

Comment:
In these integrations part of the monorepo it looks like this:
Example React: https://docs.sentry.io/platforms/javascript/guides/react/session-replay/
```suggestion
import * as Sentry from "@sentry/astro";

Sentry.init({
  dsn: "___PUBLIC_DSN___",

  // This sets the sample rate at 10%. You may want this to be 100% while
  // in development, then sample at a lower rate in production.
  replaysSessionSampleRate: 0.1,
  // If the entire session is not sampled, use the below sample rate to sample
  // sessions when an error occurs.
  replaysOnErrorSampleRate: 1.0,

  integrations: [new Sentry.Replay()],
```